src/StateService/MPT/MPTTrie.Find.cs

```diff
@@ -62,22 +62,30 @@ private ReadOnlySpan<byte> Seek(ref MPTNode node, ReadOnlySpan<byte> path, out M
             return ReadOnlySpan<byte>.Empty;
         }
 
-        public IEnumerable<(TKey Key, TValue Value)> Find(ReadOnlySpan<byte> prefix)
+        public IEnumerable<(TKey Key, TValue Value)> Find(ReadOnlySpan<byte> prefix, byte[] from = null)
         {
             var path = ToNibbles(prefix);
+            if (from is null) from = Array.Empty<byte>();
+            if (0 < from.Length)
+            {
+                if (!from.AsSpan().StartsWith(prefix))
+                    throw new InvalidOperationException("invalid from key");
+                from = ToNibbles(from[prefix.Length..].AsSpan());
+            }
             path = Seek(ref root, path, out MPTNode start).ToArray();
-            return Travers(start, path)
+            return Travers(start, path, from)
                 .Select(p => (FromNibbles(p.Key).AsSerializable<TKey>(), p.Value.AsSerializable<TValue>()));
         }
 
-        private IEnumerable<(byte[] Key, byte[] Value)> Travers(MPTNode node, byte[] path)
+        private IEnumerable<(byte[] Key, byte[] Value)> Travers(MPTNode node, byte[] path, byte[] from)
```

Comment:
Maybe we can receive a `CancellationToken` and break the iteration if the connection died?

Reply:
Currently rpc method can't get `HttpContext`. I'm afraid this change will affect all rpc method.
Maybe I can create another pr.
Do you have any suggestion about get http context here without influence on other rpc methods?

Reply:
We should do this in a separate PR to address #629 generally, not just for state service

Reply:
> We should do this in a separate PR to address #629 generally, not just for state service

Agree.

Reply:
Done.